### PR TITLE
Sponsor detail pages

### LINKED ISSUE #177: New page: /sponsors-detail
- [ ] create page `/sponsors-detail`

### PR DESCRIPTION
**Pull request by lukaszklis**
Fixes #177

### CODE REVIEW

File: templates/pages/sponsor.html.njk [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      <figure class="sponsor-details-logo">
+        <img src="{{ contents.images.sponsor[sponsor.image.filename].url }}"
+             alt="{{ sponsor.name }}">
+      </figure>
+      <div class="sponsor-details-content">
+        {{ page.html }}
+      </div>
+      <a href="{{ sponsor.link }}"
+         target="_blank"
+         class="sponsor-details-link">
```
Comment (cramforce) [inline templates/pages/sponsor.html.njk]: `rel="nofollow"`

File: templates/pages/sponsor.html.njk [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+{% block content %}
+  <div class="grid">
+    <div class="grid-item-center">
+      <a href="{{ contents['sponsors.json'].url }}"
+         class="sponsor-details-back-link">
+        &lt; back
+      </a>
+      <figure class="sponsor-details-logo">
+        <img src="{{ contents.images.sponsor[sponsor.image.filename].url }}"
```
Comment (cramforce) [inline templates/pages/sponsor.html.njk]: Lets add link (with nofollow)

File: templates/pages/sponsors.html.njk [hunk 3]
```diff
@@ -38,9 +38,7 @@
 
               <div class="sponsor sponsor-{{ sponsorsType }}"
                   id="{{ sponsor.id }}">
-                <a href="{{ sponsor.link }}"
```
Comment (cramforce) [inline templates/pages/sponsors.html.njk]: Lets keep the old behavior if `sponsor.html` is empty!

**Comment by cramforce** [comment]
Is this ready to merge?

**Comment by lukaszklis** [comment]
I'd say so, yes.

**Pull request merged**